packages/schematics/angular/application/other-files/app.module.ts

```diff
@@ -2,20 +2,15 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 <% if (routing) { %>
 import { AppRoutingModule } from './app-routing.module';<% } %>
-<% if (serviceWorker) { %>
-import { ServiceWorkerModule } from '@angular/service-worker';<% } %>
 import { AppComponent } from './app.component';
-<% if (serviceWorker) { %>
-import { environment } from '../environments/environment';<% } %>
 
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
     BrowserModule<% if (routing) { %>,
-    AppRoutingModule<% } %><% if (serviceWorker) { %>,
-    ServiceWorkerModule.register('/ngsw-worker.js', { enabled: environment.production })<% } %>
```

Comment:
Is service worker registration somewhere else now?

Reply:
Not in this PR, but it is in the PWA PR #470